packages/aws-cdk-lib/aws-appsync/README.md

```diff
@@ -519,16 +519,18 @@ the ability to add data sources and resolvers through a `IGraphqlApi` interface.
 ```ts
 declare const api: appsync.GraphqlApi;
 declare const table: dynamodb.Table;
+declare const role: iam.Role;
 const importedApi = appsync.GraphqlApi.fromGraphqlApiAttributes(this, 'IApi', {
   graphqlApiId: api.apiId,
   graphqlApiArn: api.arn,
 });
 importedApi.addDynamoDbDataSource('TableDataSource', table);
+importedApi.grantMutation(role, 'aMutation')
 ```
 
 If you don't specify `graphqlArn` in `fromXxxAttributes`, CDK will autogenerate
-the expected `arn` for the imported api, given the `apiId`. For creating data
-sources and resolvers, an `apiId` is sufficient.
+the expected `arn` for the imported api, given the `apiId`. sources, creating
+resolvers, and granting IAM permissions, an `apiId` is sufficient.
```

Comment:
Let's instead add a section [here](https://github.com/aws/aws-cdk/blob/de0a9e218eeccbe4f685e45f93e25a250733dc51/packages/aws-cdk-lib/aws-appsync/README.md?plain=1#L670):
```
### Permissions on imported APIs

You can use the `grant` functions on imported GraphQL APIs as well:

```ts
declare const api: appsync.GraphqlApi;
declare const role: iam.Role;

const importedApi = appsync.GraphqlApi.fromGraphqlApiAttributes(this, 'IApi', {
  graphqlApiId: api.apiId,
  graphqlApiArn: api.arn,
});

// For generic types
importedApi.grantMutation(role, 'updateImportedExample');

// For custom types and granular design
importedApi.grant(role, appsync.IamResource.ofType('Mutation', 'updateImportedExample'), 'appsync:GraphQL');
```
Feel free to improve it further.